cypress/e2e/gui.cy.js

```diff
@@ -0,0 +1,145 @@
+/// <reference types = "cypress" />
+
+describe('EngageSphere gui test cases', () => {
+beforeEach(() => {
+  Cypress.on('window:before:load', window => {
+    window.document.cookie = 'cookieConsent=accepted'
+  })
+  cy.visit('/')
+})
+
+    it('Keeps the filters when coming back from the customer details view ', () => {
+      cy.get('select[data-testid="size-filter"]')
+       .should('be.visible')
+       .select('Medium')
+      cy.contains('button', "View")
+       .click()
+      cy.contains('button', "Back")
+       .click()
+      cy.get('select[data-testid="size-filter"]')
+       .should('have.value', 'Medium')
+    })
+    it('It goes back to the customers list when clicking the "Back" button *', () => {
```

Comment:
```suggestion
    it('goes back to the customers list when clicking the "Back" button *', () => {
```

Que tal remover o `It` dos testes visto que else já estão dentro de um bloco `it`?

> O mesmo vale para outros testes onde isso se repete.